tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md

```diff
@@ -38,59 +38,61 @@ To create service instances for business process management services, you need t
 
     >![Account setup](02_Foundation20Onboarding_Processing.png)
 
-3. From your global account page, choose the `trial` tile to access your subaccount.
+3. From your global account page, choose the **Recipe** from left-hand navigation. Among the available recipes, click **Start Recipe** of **Set up account for Workflow Management**.
 
-    !![Enter Trial Subaccount](entertrialaccount.png)
+    !![Start Recipe](startrecipe_2.png)
 
-4. Click **Entitlements** from left panel to check if you have all the below listed entitlements in your subaccount. These entitlements are needed to run other tutorials of this mission.
+4. Recipe will be started with pre-configured steps.
```

Comment:
`**Recipe**` ?

Reply:
Its Recipe option that appears in the cockpit.

Reply:
Sorry, my comment was targeting  the word's formatting.